### PR TITLE
Removes Age-weighted RNG for Rolling Command Roles

### PR DESCRIPTION
**Pull request by SleepyGemmy**
title. people shouldn't have less chance to be able to play their wanted command role because they didn't go for the "ideal" character age.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

File: code/controllers/subsystems/job.dm [hunk 1]
```diff
@@ -159,31 +159,14 @@ SUBSYSTEM_DEF(jobs)
...
-				else if(age > (ideal_job_age - 10)) // Perfect
-					weightedCandidates[V] = 10
-				else if(age > (min_job_age + 10)) // Good, but on the young side
-					weightedCandidates[V] = 6
-				else if(age >= min_job_age) // Too young
-					weightedCandidates[V] = 3
-				else
-					if(candidates.len == 1) // There's only one option
-						weightedCandidates[V] = 1
+				weightedCandidates[V] = 1
```
Comment (SleepyGemmy) [inline code/controllers/subsystems/job.dm:169]: not the cleanest solution. maybe i could `pick()` straight from the `candidates` list?

**Comment by granodd** [comment]
I wasn't even aware this was a mechanic.

**Comment by OolongCow** [comment]
> I wasn't even aware this was a mechanic.

I'm pretty sure most people didn't.

**Review by Geevies — APPROVED** [review]

**Pull request merged**